src/HotChocolate/Core/src/Types/Configuration/AggregateTypeInterceptor.cs

```diff
@@ -54,11 +55,19 @@ public void SetInterceptors(IReadOnlyCollection<object> interceptors)
 
     internal override void InitializeContext(
         IDescriptorContext context,
+        TypeInitializer typeInitializer,
+        TypeRegistry typeRegistry,
+        TypeLookup typeLookup,
         TypeReferenceResolver typeReferenceResolver)
     {
         foreach (TypeInterceptor interceptor in _typeInterceptors)
         {
-            interceptor.InitializeContext(context, typeReferenceResolver);
+            interceptor.InitializeContext(
```

Comment:
We need to have a look at this again ... do we need all of these internal services?